README.md

```diff
@@ -1,3 +1,29 @@
 # vault2env-js
 
-`vault2env --token=[vault-token] --addr=[vault-addr] --path=secret/path/to/envs`
\ No newline at end of file
+`vault2env` let's you easily load env variables from your Vault instance and
+save them in `.env` file
```

Comment:
Please try to avoid line-breaks in the middle of the sentence. Use soft-wrap in your IDE instead...